tests/integration/targets/s3_object/meta/main.yml

```
@@ -1,2 +1,6 @@
 dependencies:
   - setup_remote_tmp_dir
+  # required for s3.get_object_attributes
+  - role: setup_botocore_pip
+    vars:
+      boto3_version: "1.22.0"
```

```suggestion
      boto3_version: "1.22.0"
```

a) Please use botocore versions this is where the API definitions live.
b) I don't see this requirement documented.

---

a) Fixed
b) not sure if I understand the point correctly, the `s3.get_object_attributes()` was added in `botocore 1.24.7` and `boto3 1.22.0`.

---

Sorry for the delayed response.

If the module requires a specific version of botocore, then that requirement should be in the module's documentation.  I'm missing where this is called out in the s3_object_info module.

---

@tremble added https://github.com/ansible-collections/amazon.aws/pull/977/files#diff-c19ed6d5998f0666b7a49cc21f1da8909f61a4c66f4a8835fd2704ab3542126aR638-R640